### PR TITLE
Add a function to get total disk cache size

### PR DESCRIPTION
**Pull request by vadymmarkov**
Fixes https://github.com/hyperoslo/Cache/issues/76

### CODE REVIEW

File: Source/Shared/Cache/BasicHybridCache.swift [hunk 1]
```diff
@@ -34,9 +34,16 @@ public class BasicHybridCache {
...
 }
 
-// MARK: - Data protection
+// MARK: - Disk cache
 
 public extension BasicHybridCache {
+  /**
+   Calculates total disk cache size
+   */
+  func totalDiskSize() throws -> UInt64 {
```
Comment (onmyway133) [inline Source/Shared/Cache/BasicHybridCache.swift]: You need this property @vadymmarkov ?

Comment (vadymmarkov) [inline Source/Shared/Cache/BasicHybridCache.swift]: What do you mean? @onmyway133

Comment (onmyway133) [inline Source/Shared/Cache/BasicHybridCache.swift]: Just read the issue and that is somebody that needs it. I thought we need it for something

**Comment by onmyway133** [comment]
🚀

File: Source/Shared/Cache/BasicHybridCache.swift (hunk 1 above)
Comment (vadymmarkov) [inline Source/Shared/Cache/BasicHybridCache.swift]: Yeah, I've never needed that.

**Comment by yukirin000** [comment]
it isn't **public** for now 🌝

**Pull request merged**